### PR TITLE
[MIG] partner_contact_department: Migration to 10.0

### PR DESCRIPTION
**Pull request by chienandalu**
- Minor migration tweaks

cc @Tecnativa

### CODE REVIEW

**Review by yajo — APPROVED** [review]
Code OK, please squash by logical changeset and author.

**Review by rafaelbn — APPROVED** [review]
Tested functionally in trunbot

File: partner_contact_department/README.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
```
Comment (rafaelbn) [inline partner_contact_department/README.rst]: pedro.baeza@tecnativa.com

File: partner_contact_department/README.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+* Antonio Espinosa <antonioea@antiun.com>
```
Comment (rafaelbn) [inline partner_contact_department/README.rst]: antonio.espinosa@tecnativa.com

File: partner_contact_department/README.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+* Rafael Blasco <rafabn@antiun.com>
```
Comment (rafaelbn) [inline partner_contact_department/README.rst]: rafael.blasco@tecnativa.com

File: partner_contact_department/README.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+* Jairo Llopis <yajo.sk8@gmail.com>
```
Comment (rafaelbn) [inline partner_contact_department/README.rst]: jairo.llopis@tecnativa.com

**Review by rafaelbn — APPROVED** [review]

**Comment by chienandalu** [comment]
Please, @rafaelbn review last changes to rebase and merge

**Review by luismontalba — APPROVED** [review]
Its OK

File: base_location_nuts/__manifest__.py [hunk 5]
```diff
@@ -1,32 +1,14 @@
...
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2015 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# Copyright 2017 David Vidal <david.vidal@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     'name': 'NUTS Regions',
```
Comment (yajo) [inline base_location_nuts/__manifest__.py]: Why is `base_location_nuts` included in this PR?

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by chienandalu** [comment]
@Yajo Sorry, I merged two branches :sweat: fixed!

File: partner_contact_department/models/res_partner.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    name = fields.Char(required=True, translate=True)
+    parent_id = fields.Many2one(
+        "res.partner.department",
+        "Parent department",
+        ondelete='restrict')
+    child_ids = fields.One2many(
+        "res.partner.department",
+        "parent_id",
+        "Child departments",
+        oldname="children")
```
Comment (yajo) [inline partner_contact_department/models/res_partner.py]: You can drop the oldname if it was already there in the last version.

**Review by yajo — APPROVED** [review]
code review

File: partner_contact_department/models/res_partner.py (hunk 6 above)
Comment (chienandalu) [reply partner_contact_department/models/res_partner.py]: Done!

**Comment by rafaelbn** [comment]
Please @chienandalu squash for merge thanks!

**Pull request merged**